src/modules/bullsCowsCalculator.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const bullsCowsCalculator = (randomDigits, userDigits) => {
+  const userDigitsToArray = userDigits.split('');
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < userDigitsToArray.length; i++) {
+    const digitIndex = randomDigits.indexOf(+userDigitsToArray[i]);
+
+    if (digitIndex !== -1) {
+      if (digitIndex === i) {
+        bulls++;
+      } else {
+        cows++;
+      }
+    }
+  }
+
+  const bullsMessage = bulls > 1 || bulls === 0 ? 'bulls' : 'bull';
+  const cowsMessage = cows > 1 || cows === 0 ? 'cows' : 'cow';
+
+  const message = `${bulls} ${bullsMessage} and ${cows} ${cowsMessage}`;
+
+  if (bulls === 0 && cows === 0) {
+    return -1;
+  }
+
+  if (bulls === 4) {
+    return 1;
+  }
+
+  return message;
```

Comment:
as for me it is better to return amount of bulls and cows

Reply:
fixed it, moved message logic to app and return object form this module